libraries/c_sdk/standard/https/src/iot_https_client.c

```diff
@@ -1792,6 +1792,10 @@ static IotHttpsReturnCode_t _parseHttpsMessage( _httpParserInfo_t * pHttpParserI
     const char * pHttpParserErrorDescription = NULL;
     http_parser * pHttpParser = &( pHttpParserInfo->responseParser );
 
+    /* Disable -Wunused-but-set-variable for local variables used for logging. */
+    ( void ) parsedBytes;
+    ( void ) pHttpParserErrorDescription;
+
```

Comment:
Why do we have these variables if they are not being used. The appropriate fix seems to be to delete the dead code?

Reply:
These variables are used in the logging macros